templates/docs/examples/layouts/global-nav.css

```diff
@@ -0,0 +1,831 @@
+// FOR DEMO PURPOSES ONLY:
+// this is a mockup of global nav for testing,
+// based on global-nav@2.4.1
+
+// stylelint-disable
```

Comment:
Not a valid comment, and I'm not sure if needed - I don't know if our linter check CSS files as well?

Reply:
No it doesn't, this is copy paste naughtiness - fixed